### DISCUSSION TITLE
Can.t find: Breakpoint->Hardware, Write->Byte

### DISCUSSION THREAD

**Discussion by Tony-Moonman** [x64dbg/x64dbg#3407] [Q&A] [answered]
I'm brand new to _this_ learning by doing. I've been interested all my life but never had the time. Now 47 years old I guess I got that feeling most of you guys got in your teen years. 
I'm currently following an kind of paint by number instruction telling me to goto Breakpoint -> Hardware, Write -> Byte.
I guess this instruction is a bit older 'cause in this version of x64dbg there's no Hardware, Write -> Byte option in the Breakpoint menu.

Can anyone explain how to make Hardware, Write -> Byte Breakpoint in x64dbg Jul 24 2024 version.

**Comment by Tony-Moonman** [comment, marked as answer]
Solved it my self.
Didn't understand I needed to click in the dump...

**Reply by mrexodia** [reply]
Alternatively you can use the [`SetHardwareBreakpoint`](https://help.x64dbg.com/en/latest/commands/breakpoint-control/SetHardwareBreakpoint.html) command.
